mwdb/web/src/commons/ui/ObjectLink.js

```diff
@@ -9,9 +9,15 @@ import Hash from "./Hash";
 export default function ObjectLink(props) {
     const auth = useContext(AuthContext);
     const objectType = mapObjectType(props.type);
+    let remotePath = "";
+    let { remote } = useParams();
+    if (remote) remotePath = `/remote/${remote}`;
```

Comment:
It can be simplified
```suggestion
    const { remote } = useParams();
    const remotePath = remote ? `/remote/${remote}`: "";
```